### PR TITLE
georgettica/initial cleaned

### PR DESCRIPTION
**Pull request by georgettica**
this PR is an adaptation of #1 and is supposed to be very minimal yet composable

to install you can use:
```
cd cadctl; GOBIN=${PWD} go install .; ./cadctl
```

and has:
- a README file
- no additional configs that is needed
- short Descriptions to all commands

some notations:
- put the cadctl to a subfolder if we need multiple things in the same repo (tekton deploy + UI)

### CODE REVIEW

**Comment by georgettica** [comment]
@rafael-azevedo who should be the author or the files?
(I wanted to add the LICENCE and it wanted a `owner`, LMK if you want something specific

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+anomaly and inform the cluster owner of the next steps to do on his belalf.
+
+## Install
+
+### cadctl
+
+to install it use the command:
+
+```shell
+go install github.com/openshift/configuration-anomaly-detection/cadctl@latest
```
Comment (georgettica) [inline README.md:15]: this is yet to be tested, but should be ok

**Review by georgettica — COMMENTED** [review]
I moved everything to `cadctl` in case we will want to add additional usaged to the `configuration-anomaly detection` later on

additionally, seems there is now prow configured just yet for this

File: cadctl/cmd/remediate/remediate.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/*
```
Comment (rafael-azevedo) [inline cadctl/cmd/remediate/remediate.go:1]: I think we can start with just the cluster-missing command for now even

Comment (georgettica) [reply cadctl/cmd/remediate/remediate.go:1]: Will move it up and change the commits

File: README.md (hunk 1 above)
Comment (georgettica) [reply README.md:15]: that worked!
great! :)

**Comment by georgettica** [comment]
closed OSD-9774

**Pull request merged**